oqs_scripts/testrun.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+import subprocess
+import os
+import time
+
+# Parameter checks already done in shellscript
+
+with open("assignments.json", "r") as f:
+   jsoncontents = f.read();
+
+onlysigoutput = False
+if len(sys.argv)>1:
+   onlysigoutput=True
+
+assignments = json.loads(jsoncontents)
+for sig in assignments:
+    print("Testing %s:" % (sig))
+    for kem in assignments[sig]:
+       # assemble testing command
+       cmd = "(echo \'GET /\'; sleep 0.2) | ../build/tool/bssl client -connect test.openquantumsafe.org:"+str(assignments[sig][kem]) + " -root-certs CA.crt 2>&1"
```

Comment:
What about something like

```python
proc = subprocess.Popen([os.path.join(build, tool, bssl), "client", 
                                     "-connect", "test.openquantumsafe.org:", str(assignments[sig][kem]),
                                     "-curves", kem,
                                     "-root-certs",  "CA.crt"]
time.sleep(0.2)
stdout, stderr = proc.communicate(input="echo \'GET /\")
```

Reply:
I tried that pattern before but never got it to work: For some reason the "GET /" simply doesn't get pushed to the server (i.e., the command `python3 oqs_scripts/testrun.py` hangs. When entering it manually it works then. I'm at a loss. Please feel free to take a look in the new revision.